### DISCUSSION TITLE
What time should I join the zoom class?

### DISCUSSION THREAD

**Discussion by Sudkorea** [irgroup-classrooms/dis17-2021#2] [Q&A]
I read all of the introduction and syllabus. Still, I could not find it.

**Comment by phschaer** [comment]
We will start tomorrow at 8am. I know, it's early in the morning. So, bring some coffee! :-)

You can find this information here at the official list of all courses at the IWS homepage: https://www.iws.th-koeln.de/vv/vv-stgsem-iws1.php?stg=DIS&sem=5&no_layout=1

All the best

Philipp Schaer
